ui/pages/build-status.js

```diff
@@ -88,6 +88,22 @@ const RepoStatusItem = ({ repo, organization = 'Mender', branch = 'master', cove
   </Stack>
 );
 
+const buildStatusColor = (status) => {
+    switch(status) {
+    case 'FAILED':
+        return 'error';
+        break;
+    case 'RUNNING':
+        return 'inprogress';
+        break;
+    case 'SUCCESS':
+        return 'success';
+        break;
+    default:
+        return '#FFA500'; // Orange
```

Comment:
to make use of the theming capabilities I would suggest to use `warning` from [here](https://mui.com/material-ui/customization/palette/) or define a color in the theme 👍